python/pyspark/pandas/frame.py

```diff
@@ -551,11 +551,11 @@ def ndim(self) -> int:
         >>> df = ps.DataFrame([[1, 2], [4, 5], [7, 8]],
         ...                   index=['cobra', 'viper', None],
         ...                   columns=['max_speed', 'shield'])
-        >>> df
+        >>> df  # doctest: +SKIP
```

Comment:
qq: Why do we need to skip this?

Reply:
The fix will be failed with panda 1.3.x (it's [current version of ci infra](https://github.com/dongjoon-hyun/ApacheSparkGitHubActionImage/blob/ac09d7a4431b209caa86090c54bfccd4d744d0ce/Dockerfile#L32)), as I mentioned in [previous comment](https://github.com/apache/spark/pull/36712#issuecomment-1140234328) , we will cleanup `# doctest: +SKIP` after we upgrade pandas to 1.4.

Reply:
BTW, here is the final test with pandas 1.4. all changes have been submited as PRs

[1] https://github.com/Yikun/spark/pull/102